front/lib/resources/data_source_view_resource.ts

```diff
@@ -125,20 +170,34 @@ export class DataSourceViewResource extends ResourceWithVault<DataSourceViewMode
     return dataSourceViews;
   }
 
-  static async listByWorkspace(auth: Authenticator) {
-    const dataSourceViews = await this.baseFetch(auth);
+  static async listByWorkspace(
+    auth: Authenticator,
+    fetchDataSourceViewOptions?: FetchDataSourceViewOptions
```

Comment:
Can we only accept those options when needed, like `listByVaults`?

Reply:
this is optional and anyway implemented in baseFetch ?